sycl/doc/EnvironmentVariables.md

```diff
@@ -23,6 +23,9 @@ compiler and runtime.
 | `SYCL_ENABLE_DEFAULT_CONTEXTS` | '1' or '0' | Enable ('1') or disable ('0') creation of default platform contexts in SYCL runtime. The default context for each platform contains all devices in the platform. Refer to [Platform Default Contexts](extensions/supported/sycl_ext_oneapi_default_context.asciidoc) extension to learn more. Enabled by default on Linux and disabled on Windows. |
 | `SYCL_RT_WARNING_LEVEL` | Positive integer | The higher warning level is used the more warnings and performance hints the runtime library may print. Default value is '0', which means no warning/hint messages from the runtime library are allowed. The value '1' enables performance warnings from device runtime/codegen. The values greater than 1 are reserved for future use. |
 | `SYCL_USM_HOSTPTR_IMPORT` | Integer | Enable by specifying non-zero value. Buffers created with a host pointer will result in host data promotion to USM, improving data transfer performance. To use this feature, also set SYCL_HOST_UNIFIED_MEMORY=1. |
+| `SYCL_REDUCTION_PREFERRED_WORKGROUP_SIZE_CPU` | Positive integer | Controls the preferred work-group size for reductions on CPU. If `info::device::max_work_group_size` on a device on which a reduction is being enqueued on is less than the value of this environment variable, that value is used instead. If this environment variable is set to 0 or an invalid value, a `sycl::exception` with `sycl::errc::invalid` is thrown during submission of the kernel with a reduction on CPU. If this is unset the preferred work-group size for reductions on CPU is implementation defined. |
```

Comment:
Should we have one env var SYCL_REDUCTION_PREFERRED_WORKGROUP_SIZE with values format similar to SYCL_DEVICE_FILTER instead of 3 separate ones?

Reply:
I have made it a single (`SYCL_REDUCTION_PREFERRED_WORKGROUP_SIZE`) env var with a similar syntax to `SYCL_DEVICE_FILTER`. Is this similar to what you had in mind?